sandbox/apparmor/apparmor_test.go

```diff
@@ -645,11 +645,8 @@ func (s *apparmorSuite) TestPromptingSupported(c *C) {
 	defer restore()
 
 	supported, reason := apparmor.PromptingSupported()
-	// TODO: change this once snapd supports prompting
-	c.Check(supported, Equals, false)
-	c.Check(reason, Equals, "requires newer version of snapd")
-	// c.Check(supported, Equals, true)
-	// c.Check(reason, Equals, "")
+	c.Check(supported, Equals, true)
+	c.Check(reason, Equals, "")
```

Comment:
Doing this here means the capstone PR can be closed: https://github.com/canonical/snapd/pull/14318/commits/b9653eaff9e35c4fd14e094ae64ec492eebe4eb8

I agree it's better to do this now, that way we can test more thoroughly now and later in the PR chain. Only worry is if someone running edge manually enables prompting, but that's unlikely.